packages/cli/test/unit/commands/container/run.unit.test.ts

```diff
@@ -0,0 +1,71 @@
+import {stdout, stderr} from 'stdout-stderr'
+import Cmd from '../../../../src/commands/container/run'
+import runCommand from '../../../helpers/runCommand'
+import * as sinon from 'sinon'
+import {expect} from 'chai'
+import * as DockerHelper from '../../../../src/lib/container/docker_helper'
+import expectOutput from '../../../helpers/utils/expectOutput'
+
+describe('container run', () => {
+  let sandbox: sinon.SinonSandbox
+
+  beforeEach(() => {
+    process.env.HEROKU_API_KEY = 'heroku_token'
+    sandbox = sinon.createSandbox()
+  })
+  afterEach(() => sandbox.restore())
+
+  it('requires a process type', async () => {
+    await runCommand(Cmd, [
+      '--app',
+      'testapp',
+    ]).catch((error: any) => {
+      expect(error.message).to.contain('Requires one process type')
+      expect(stdout.output, 'to be empty')
+    })
```

Comment:
Optional but avoids the use of `any`.
```suggestion
    ]).catch((error) => {
      const {message} = error as {message: string}
      expect(message).to.contain('Requires one process type')
      expect(stdout.output, 'to be empty')
    })
```